### PR TITLE
[Edu] When in an active session, if I refresh the app it calls the st…

### PR DESCRIPTION
**Pull request by OfirZinkerz**
…udent automatically

### CODE REVIEW

File: src/components/activePanel/activePanel.directive.js [hunk 1]
```diff
@@ -162,6 +162,34 @@
...
+
+                                var dataToSave = {};
+                                dataToSave[liveSessionData.guid] = 'disable';
+                                // Save data to sessionStorage
+                                sessionStorage.setItem('liveSessionAutoCall', JSON.stringify(dataToSave));
+                            }
+                        } else if (liveSessionData.status === LiveSessionStatusEnum.ENDED.enum) {
+                            // Remove saved data from sessionStorage
+                            sessionStorage.removeItem('liveSessionAutoCall');
+                        }
```
Comment (assafshp) [inline src/components/activePanel/activePanel.directive.js]: what about toggleAutoCallEnum.DISABLE.enum : toggleAutoCallEnum.ACTIVATE.enum ?

File: src/components/activePanel/activePanel.directive.js [hunk 2]
```diff
@@ -162,6 +162,34 @@
...
+                    function handelAutocall(isOffline, receiverId, liveSessionData) {
+                        if (liveSessionData.status === LiveSessionStatusEnum.CONFIRMED.enum){
+                            scope.d.callBtnModel = {
+                                isOffline: isOffline,
+                                receiverId: receiverId,
+                                toggleAutoCall: toggleAutoCallEnum.DISABLE.enum
+                            };
+
+                            // Get saved data from sessionStorage
+                            var autoCallData = sessionStorage.getItem('liveSessionAutoCall');
```
Comment (assafshp) [inline src/components/activePanel/activePanel.directive.js]: son't we have a dedictaed service for sessionStorage ?

Comment (assafshp) [inline src/components/activePanel/activePanel.directive.js]: Is it sync call ? promise?

Comment (OfirZinkerz) [inline src/components/activePanel/activePanel.directive.js]: $window was add before sessionStorage

File: src/components/activePanel/activePanel.directive.js (hunk 1 above)
Comment (OfirZinkerz) [inline src/components/activePanel/activePanel.directive.js]: fixed

**Pull request merged**